utils/rowgroup/rowaggregation.cpp

```diff
@@ -4180,7 +4079,14 @@ void RowAggregationUMP2::doAvg(const Row& rowIn, int64_t colIn, int64_t colOut,
     if (cnt == 0)
     {
         fRow.setLongDoubleField(valIn, colOut);
-        fRow.setUintField(rowIn.getUintField(colIn + 1), colAux);
+        if (merge)
```

Comment:
What is the difference b/w these two branches?

Reply:
If it was used while update the aggregated row with the input row it just increments the aggregated value by 1, otherwise it sums the target aggregated value with the source aggregated value.

Reply:
In this particular case the statements in both scopes are identical. Or I am wrong?